project/scripts/genDocsScalaLang

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -e
+shopt -s extglob # needed for rm everything but x
+echo "Working directory: $PWD"
+
+GENDOC_EXTRA_ARGS=$@
+GIT_HEAD=$(git rev-parse HEAD) # save current head for commit message in gh-pages
+PREVIOUS_SNAPSHOTS_DIR="$PWD/../prev_snapshots"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >& /dev/null && pwd)"
+SITE_OUT_DIR="$PWD/docs/_site"
+
+DOCS_SCALA_LANG_DIR="$PWD/docsScalaLang"
+
+rm -rf $DOCS_SCALA_LANG_DIR
+mkdir -pv $DOCS_SCALA_LANG_DIR
+git clone "https://github.com/scala/docs.scala-lang.git" $DOCS_SCALA_LANG_DIR
+
+SBT="$SCRIPT_DIR/sbt"
+mkdir -pv $SITE_OUT_DIR
+env "scaladoc.projectFormat=md" "$SBT" "scaladoc/renderScaladocScalajsToFile $DOCS_SCALA_LANG_DIR/scripts/scaladoc-scalajs.js $DOCS_SCALA_LANG_DIR/resources/css code-snippets.css"
+"dist/target/pack/bin/scaladoc" "-d" "$SITE_OUT_DIR" "-format" "md" "-siteroot" "docs" "/dev/null"
+
+if [ ! -d "$SITE_OUT_DIR" ]; then
+  echo "Output directory did not exist: $SITE_OUT_DIR" 1>&2
+  exit 1
+fi
+
+# Copy reference and scaladoc docs
+cp -rf "$SITE_OUT_DIR/docs/reference"/* "$DOCS_SCALA_LANG_DIR/_scala3-reference"
+cp -rf "$SITE_OUT_DIR/docs/usage/scaladoc"/* "$DOCS_SCALA_LANG_DIR/_overviews/scala3-scaladoc"
+
+cp -rf "$SITE_OUT_DIR/docs/reference/contextual/motivation.md" "$DOCS_SCALA_LANG_DIR/_scala3-reference/contextual.md"
+cp -rf "$SITE_OUT_DIR/docs/reference/metaprogramming/toc.md" "$DOCS_SCALA_LANG_DIR/_scala3-reference/metaprogramming.md"
+cp -rf "$SITE_OUT_DIR/docs/resources/talks.md" "$DOCS_SCALA_LANG_DIR/scala3/talks.md"
+cp -rf "$SITE_OUT_DIR/docs/usage/getting-started.md" "$DOCS_SCALA_LANG_DIR/scala3/getting-started.md"
+cp -rf "$SITE_OUT_DIR/docs/usage/language-versions.md" "$DOCS_SCALA_LANG_DIR/_scala3-reference/language-versions.md"
+cp -rf "$SITE_OUT_DIR/docs/usage/worksheet-mode.md" "$DOCS_SCALA_LANG_DIR/_overviews/scala3-book/tools-worksheets.md"
+
+
+# Copy csses and html importing these assets
+cp -f "$SITE_OUT_DIR/styles/colors.css" "$DOCS_SCALA_LANG_DIR/resources/css/colors.css"
+cp -f "$PWD/docs/docsScalaLangResources/scaladoc-assets.html" "$DOCS_SCALA_LANG_DIR/_includes/scaladoc-assets.html"
```

Comment:
How will users of the docs.scala-lang.org repo know that these files are managed here, and not there?

Reply:
There is a comment at the top of every markdown file that informs them they were autogenerated. I wanted to add the same button for docs.scala-lang as we do have at dotty.epfl.ch with `Edit on github` button.
![obraz](https://user-images.githubusercontent.com/32793002/129875172-d38ce005-02f3-4156-9d58-0c5fa88b6665.png)

Reply:
But these are not markdown files but css files, no?

Reply:
Oh, I missed what line you are referring to. I think we could add comments on top of these files as we do with markdown files., or do you have some better ideas.